### DISCUSSION TITLE
API Gateway Timeout Increase

### DISCUSSION THREAD

**Discussion by divy6413** [serverless/serverless#12631] [Q&A] [answered]
Hi ,
I want to increase timeout of REST APIs in api gateway.
We have our quota increased from 29s and option is available in Console at API->Resources->Integration request->Timeout.
But I could not find anything in serverless docs to configure the same for REST APIs.
Is this possible via serverless, if yes how to do it?

**Comment by nacholupotti** [comment]
Hey I have the exact same question, have you find a solution?

**Comment by michaelbrant** [comment]
anyone find a solution?

**Comment by czubocha** [comment, marked as answer]
The ability to set the AWS API Gateway integration timeout has been released in [v4.4.13](https://github.com/serverless/serverless/releases/tag/v4.4.13). You can find the docs [here](https://www.serverless.com/framework/docs/providers/aws/events/apigateway#timeout).
Feel free to share any further issues, feature requests, or feedback!
